### ISSUE TITLE
Soft hyphen (00AD) is invisible

### ISSUE THREAD

**Issue by drs-p** [CatharsisFonts/Ysabeau#67]
The soft hyphen in the Ysabeau fonts is invisible and has width zero. That is a problem in a Tagged PDF as soft hyphens should be used for hyphenated words to improve accessibility and screen reading. See the discussion at this issue: https://github.com/latex3/tagging-project/issues/967, and also https://jkorpela.fi/shy.html#pdf for a discussion about the soft hyphen in general and its use in PDF specifically.

**Comment by drs-p** [comment]
On second thoughts, there is some discussion whether this is the right thing to do (see the ongoing thread at latex3/tagging-project#967). So I'll close this issue for now, sorry for the noise!

**Comment by cmahte** [comment]
The core Unicode Specification page 327 (6.2.4 - Hyphens and Dashes)  and  page 1038 (23.2.1 Line and Word Breaking) 

help to explain what the purpose of the various breaks (invisble and visible) and visible characters related to hyphenation are for. 

https://www.unicode.org/versions/Unicode16.0.0/UnicodeStandard-16.0.pdf

Some of what you pointed to infers that the PDF engine designer converts the soft hyphen to a hyphen when it occurs at a line ending... but I didn't see it in the immediate vicinity of your link.  

But soft hyphens are supposed to be invisible. I've seen rare fonts draw them visible with zero width or even visible with a width, but these are intended for screens during design phase, and not reading mode screens or print use. 

Good Luck!
